cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -41,57 +36,83 @@ context("Researcher uses permission forms", () => {
         pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
-            false, TEST_PROJECT_NAME);
+            TEST_PROJECT_NAME);
 
+        cy.log('Confirm that the expected Project name is selected');
+        pf.verifyCreateEditPermissionFormProjectName(TEST_PROJECT_NAME);
+    
         cy.log('Create more Permission Forms, to be used to test the Archive, Unarchive & Delete buttons');
         pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
-            false, TEST_PROJECT_NAME);
+            TEST_PROJECT_NAME);
         pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
-            false, TEST_PROJECT_NAME);
-        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'4', 'learn.portal.staging.'+TEST_PERMISSION_FORM_URL,
-            false, TEST_PROJECT_NAME);
+            TEST_PROJECT_NAME);
 
-        // TODO: Replace this unnecessary line(s) with a check that the Project name displayed is the one that we expect
-        // cy.log("Select a Project from the dropdown (on the 'Create New Permission Form' tab)");
-        // pf.selectCreateManagePermissionsProject(TEST_PROJECT_NAME);
+        cy.log('Create still more Permission Forms, to be used to test the Manage Student Permissions tab');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'4', 'learn.portal.staging.'+TEST_PERMISSION_FORM_URL,
+            TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'5', 'www.'+TEST_PERMISSION_FORM_URL,
+            TEST_PROJECT_NAME);
+    
+        cy.log('Confirm that the expected Project name is still selected');
+        pf.verifyCreateEditPermissionFormProjectName(TEST_PROJECT_NAME);
 
-        cy.log('Test the links on the Permission Forms table, starting with URL text, Archive, and Unarchive');
+        cy.log('Test the links on the Permission Forms table, starting with the URL text, ARCHIVE, and UNARCHIVE');
         pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
         pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
         pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
 
-        // TODO: fix clicking on archived rows, then uncomment these:
-        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'2');
-        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'3');
-        // pf.clickPermissionFormUnarchive(TEST_PERMISSION_FORM_NAME+'3');
+        pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'2');
+        pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'3');
+        pf.clickPermissionFormUnarchive(TEST_PERMISSION_FORM_NAME+'3');
```

Comment:
After we archive the permission form it will be displayed in the disabled view in the portal. i.e., that archived permission form will be displayed with the class **isArchived**. Shall we add coverage to verify that scenario?